### ISSUE TITLE
[8.0.0 and below] Uninformative Orange bar while posting laws when not yet having reached the citizenship requirements

### ISSUE THREAD

**Issue by bleckman74** [StrangeLoopGames/EcoIssues#10020]
![untitled](https://user-images.githubusercontent.com/25288501/48522632-5c14f700-e847-11e8-88fa-36375ae236ca.png)

when i go to make a Law get:

error [object: Object]

**Comment by bleckman74** [comment]
its laws, Denogfraics,  Everything

**Comment by D3nnis3n** [comment]
I got that issue with districts only and only once. Cannot reproduce reliably otherwise.

@bleckman74 Could you attach your world save? You can upload it to dropbox or google drive for free.

Related

**Comment by Deantwo** [comment]
Before I passed requirement for Citizenship law, I was getting a simlir error message when attempting to run for office.

**Comment by msgerbs** [comment]
I don't see the error message at all on Firefox, just an orange bar.

**Comment by elfl0rd** [comment]
https://github.com/StrangeLoopGames/EcoIssues/issues/10144
https://github.com/StrangeLoopGames/EcoIssues/issues/10154
;)

**Comment by D3nnis3n** [comment]
One of them is mainly about lag and the other one has another cause for the object issue, so those are no real duplicates.

**Comment by Deantwo** [comment]
Duplicate of #10084

**Comment by D3nnis3n** [comment]
That one is already closed.

**Comment by Deantwo** [comment]
It is, but it is not solved, the player just closed it...
Either way, just wanted to reference it.
